pkg/envoy/xds_server.go

```diff
@@ -79,16 +79,9 @@ const (
 	adminListenerName     = "envoy-admin-listener"
 )
 
-type Listener struct {
+type ListenerCount struct {
```

Comment:
can this type be unexported?

Reply:
or completely get rid of the struct and use the type `uint` of its only field `count` directly as the value of the `listenerCount` map

Reply:
I considered this but at the time decided that updating the pointed uint is simpler than updating the map, but I agree, overall this is simpler, changing...

Reply:
Changed.